<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="labelSlika">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>papa.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>309</width>
     <height>145</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="4">
     <widget class="QLineEdit" name="lineIme"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Godine</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Ime</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="3">
     <widget class="QLineEdit" name="lineGodine"/>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QLineEdit" name="linePasos"/>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>ID_SJEDISTA</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Broj Pasosa</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="3">
     <widget class="QLineEdit" name="linePrezime"/>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="lineID">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Prezime</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Rezervisi </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
